packages/mui-utils/tsconfig.build.json

```diff
@@ -3,6 +3,7 @@
   // Actual .ts source files are transpiled via babel
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "composite": true,
```

Comment:
Almost the only package with this flag on https://www.typescriptlang.org/docs/handbook/project-references.html#composite.

Reply:
We should aim to introduce this flag on all our projects (after we deal with circular references) to speed up the build times.

Reply:
Oops, I wrote a typo: ~with~ without.